packages/react-form-renderer/src/form-renderer/field-provider.js

```diff
@@ -28,7 +28,9 @@ class FieldProvider extends Component{
           { ...fieldsProps }
           input={{
             ...input,
-            onChange: (...args) => enhancedOnChange(dataType, onChange, ...args),
+            onChange: (...args) => {
```

Comment:
I don’t think this will cause any issue, but using the block you removed the return statement.

I would not use this block, at least for code consistency.